code/grant/pokedex/pokedex/management/commands/load_pokemon.py

```diff
@@ -0,0 +1,1953 @@
+import requests
+
+from django.core.management.base import BaseCommand
+
+from pokedex.models import Pokemon, PokemonType
+
+import json
+
+# f = {
```

Comment:
When this lab is complete, can this be removed when it's submitted? It makes it difficult to review the code. Thanks!

Reply:
Am i removing the whole file or just these 4 lines?